src/Elliptic/Systems/Xcts/Tags.hpp

```diff
@@ -121,5 +125,112 @@ struct ShiftStrain : db::SimpleTag {
   using type = tnsr::ii<DataType, Dim, Frame>;
 };
 
+/*!
+ * \brief The conformal longitudinal operator applied to the shift excess
+ * \f$(\bar{L}\beta^\mathrm{excess})^{ij}\f$
+ *
+ * This quantity can be used as the "flux" for the momentum constraint in
+ * formulations of the XCTS equations, because the principal part of the
+ * momentum constraint is essentially the divergence of this quantity.
+ */
+template <typename DataType, size_t Dim, typename Frame>
+struct LongitudinalShiftExcess : db::SimpleTag {
+  using type = tnsr::II<DataType, Dim, Frame>;
+};
+
+/*!
+ * \brief The conformal longitudinal operator applied to the background shift
+ * vector minus the time derivative of the conformal metric
+ * \f$(\bar{L}\beta^\mathrm{background})^{ij} - \bar{u}^ij\f$
+ *
+ * Note that the time derivative of the conformal metric has its indices raised
+ * as \f$\bar{u}^ij = \bar{\gamma}^{ik}\bar{\gamma}^{jl}
+ * \partial_t\bar{\gamma}_{kl}\f$, i.e. by the conformal metric, as usual for
+ * conformal quantities.
+ *
+ * \see `Xcts::Tags::ShiftBackground`
+ */
+template <typename DataType, size_t Dim, typename Frame>
+struct LongitudinalShiftBackgroundMinusDtConformalMetric : db::SimpleTag {
+  using type = tnsr::II<DataType, Dim, Frame>;
+};
+
+/*!
+ * \brief The conformal longitudinal operator applied to the shift vector minus
+ * the time derivative of the conformal metric, squared:
+ * \f$\left((\bar{L}\beta)^{ij} - \bar{u}^ij\right)
```

Comment:
since this is taking up 2 lines in C++, maybe make it an `\f{align*}` type equation? Up to you :)

Reply:
I tried, though IMO the inline math looks better when rendered:

<img width="531" alt="Bildschirmfoto 2021-03-08 um 10 39 56" src="https://user-images.githubusercontent.com/746230/110303676-f130f080-7ffa-11eb-913d-a8de6c246a07.png">
<img width="515" alt="Bildschirmfoto 2021-03-08 um 10 42 03" src="https://user-images.githubusercontent.com/746230/110303680-f3934a80-7ffa-11eb-9697-71eb562f1469.png">

Reply:
Okay, either is fine with me :)